requirements.txt

```diff
@@ -2,3 +2,5 @@ django
 django-extensions
 django-bootstrap3
 psycopg2
+numpy
```

Comment:
panel/scripts.py line 51. Numpy module got option that exclude weekends from time period